bin/methods/install.js

```diff
@@ -0,0 +1,22 @@
+const cwd = process.cwd()
+const log = require('log-cb')
+const npm = require('npm')
```

Comment:
Using all of npm is usually a bad idea.  Might want to find smaller functions like `npm-install`.